apps/extension/webpack.config.ts

```diff
@@ -67,7 +67,6 @@ const config: webpack.Configuration = {
   },
   module: {
     rules: [
-      { test: /\.wasm/ },
```

Comment:
When you say `empty rule is no-op`, I think that means without specifying a loader it doesn't do anything? I guess the `asyncWebAssembly` setting is handling everything for us already?